python/generate_syntax_highlightining.py

```diff
@@ -120,32 +120,46 @@ def write_language_json(name, scope_name, filename,
                     support=support, strings=strings,
                     comment=comment, variable=variable)
 
-## Glossary files.  Reference:
+## Glossary files.  References:
 ## http://oracc.museum.upenn.edu/doc/help/glossaries/index.html
+## https://build-oracc.museum.upenn.edu/doc/help/glossaries/cbd2/index.html
 keywords = {}
-entry_keywords = ("bases|form|sense|note|inote|equiv|isslp|bib|parts|bff|" +
-                  "collo|prop")
+entry_keywords = "alias|parts|bff" # NOTE: bff not documented
```

Comment:
```suggestion
entry_keywords = "alias|parts|bff" # NOTE: bff not officially supported in newer version
```
If I remember right?